src/flags/size.rs

```diff
@@ -19,6 +21,23 @@ pub enum SizeFlag {
     Bytes,
 }
 
+impl SizeFlag {
+    fn from_str(value: &str) -> Option<Self> {
+        match value {
+            "default" => Some(Self::Default),
+            "short" => Some(Self::Short),
+            "bytes" => Some(Self::Bytes),
+            _ => {
+                print_error!(
+                    "size can only be one of default, short or bytes, but got {}",
```

Comment:
We probably will not reach here, but I think it would be better for all the error messages to be proper like capitalizing first letter, `.` at end and some basic stuff like that. You probably might have kept it to the end, just thought I would mention so as to not forget.